content/blog/2024-05-28-announcing-810-stable.md

```diff
@@ -0,0 +1,100 @@
+    * [Google Cloud](https://wiki.almalinux.org/cloud/Google.html)
+    * [Generic Cloud/Cloud-init](https://wiki.almalinux.org/cloud/Generic-cloud-on-local.html) for all 4 architectures
+    * [OpenNebula](https://wiki.almalinux.org/cloud/OpenNebula.html) for x86_64 and AArch64 architectures
+    * [Oracle Cloud Infrastructure](https://wiki.almalinux.org/cloud/OCI.html) for x86_64 and AArch64 Instances
+* [Vagrant Boxes](https://app.vagrantup.com/almalinux):
+    * Libvirt
+    * VirtualBox
+    * Hyper-V
+    * VMWare for x86_64 and AArch64
+    * Parallels for AArch64
+* [Raspberry Pi](https://wiki.almalinux.org/documentation/raspberry-pi.html)
+* [Windows Subsystem for Linux](https://wiki.almalinux.org/documentation/wsl.html) for x86_64 and AArch64
+
+## Release Notes and More Information
+
+AlmaLinux 8.10 brings updates to security and data protection, and improvements in web-console and system roles to automate operations and ensure consistency in intricate IT settings. The release continues to enhance system availability, reliability, and recovery processes, alongside improving virtual machine snapshot functions in hybrid cloud scenarios. New system roles have been introduced to streamline the creation and administration of logical volume manager (LVM) snapshots for better data backup and recovery processes. Performance, scalability, and reliability continue to be the focus of updates in the 8.10 version to aid developers in application development and management.
+
+You can read the full release notes for this version on the wiki: [AlmaLinux OS 8.10 Release Notes](https://wiki.almalinux.org/release-notes/8.10.html).
+
+## Renewing support for upstream deprecated hardware
+
+With AlmaLinux 8.10 we have also differentiated ourselves by meeting another request from our community for support with older hardware to help them stay on updated operating systems.
+
+The following device drivers were modified to re-add PCI IDs for hardware disabled in upstream:
+* The following device drivers were modified to re-add PCI IDs for hardware disabled in upstream:
+    * **aacraid** -  Dell PERC2, 2/Si, 3/Si, 3/Di, Adaptec Advanced Raid Products, HP NetRAID-4M, IBM ServeRAID & ICP SCSI 
+    * **be2iscsi** - Emulex OneConnect Open-iSCSI for BladeEngine 2 and 3 adapters 
+    * **be2net** - Emulex BladeEngine 2 and 3 adapters
+    * **hpsa** - HP Smart Array Controller 
+    * **lpfc** - Emulex LightPulse Fibre Channel SCSI 
+    * **megaraid_sas** - Broadcom MegaRAID SAS 
+    * **mlx4_core** - Mellanox Gen2 and ConnectX-2 adapters
+    * **mpt3sas** - LSI MPT Fusion SAS 3.0 
+    * **mptsas** - Fusion MPT SAS Host 
+    * **qla2xxx** - QLogic Fibre Channel HBA 
+    * **qla4xxx** - QLogic iSCSI HBA 
+
+For a complete list of hardware support for which was added in this release, see the  [Extended hardware support](https://wiki.almalinux.org/release-notes/8.10.html#extended-hardware-support) section in the 8.10 release notes.
```

Comment:
Same as above, the wiki URI needs to be updated prior to publishing: https://wiki.almalinux.org/release-notes/8.10-beta.html#extended-hardware-support

Reply:
Answer in previous comment)